web/locale/en_US/messages.po

```diff
@@ -1605,10 +1605,10 @@ msgstr "Mentor/Volunteer"
 msgid "Champion"
 msgstr "Champion"
 
-msgid "recurring"
-msgstr "Recurring"
+msgid "Term"
+msgstr "Term"
 
-msgid "one-off"
+msgid "One off"
```

Comment:
Actually turns out there's already a "One off" msgid lower down in the file, so @BSKERRITT is removing this one. one-off as a msgid was actually never used.